.golangci.yaml

```diff
@@ -33,9 +37,9 @@ run:
   # we confidently recognize autogenerated files.
   # If it's not please let us know.
   # "/" will be replaced by current OS file path separator to properly work on Windows.
-  # skip-files:
+  skip-files:
   #   - ".*\\.my\\.go$"
-  #   - lib/bad.go
+    - redfish/provider/common.go
```

Comment:
Please add "TODO"s so that we dont forget to re-enable these checks

Reply:
And looks like u fixed quite a few lint issues on this file. Please check if it still needs ignoring.

Reply:
there has been cyclometric issue for one of existing function.

Reply:
for specific issues, please consider [specific nolint directives](https://golangci-lint.run/usage/false-positives/#nolint-directive) as opposed to ignoring whole files